DetectedObj[] detected_obj
bool brake_status
